app/src/js/program.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+  "use strict";
+
+  // Mobile navigation toggle
+  const submenuScroll = {
+    init: function() {
+      let submenuContainer = document.getElementById("submenu-container");
```

Comment:
Volgende keer zou ik deze variabelen verdelen in het object en niet alles in de init() functie zetten.